tests.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+require 'libraries' . DIRECTORY_SEPARATOR . 'helpers.php';
+
+
+require prepare_file_path('app/Repositories/FileBaseRepository.php');
+require prepare_file_path('app/Repositories/JsonBaseRepository.php');
+require prepare_file_path('app/Repositories/UserBaseRepository.php');
+
+use App\Repositories\UserRepository;
+
+putenv('dirName=test_data_storage');
+
+testCreateValue([
+    'username' => 'username',
+    'password' => 'password'
+]);
+
+testDataCreate([
+    'username' => 'username',
+    'password' => 'password'
+]);
+
+readline();
+
+function testCreate ($dataTest)
+{
+    $exmpClass = new UserRepository();
+
+    file_put_contents('test_data_storage/users.json', '');
+
+    return $exmpClass->create($dataTest);
+
+}
+
+function testCreateValue($dataTest)
+{
+    $data = testCreate($dataTest);
+
+    if ($data === $dataTest) {
+        echo 'Success' . "\n";
+    } else {
+        echo 'Fail' . "\n";
+    }
```

Comment:
let's move this logic to the separate method `assertEquals($firstValue, $secondValue)`